WalletWasabi/Helpers/Constants.cs

```diff
@@ -12,6 +12,7 @@ public static class Constants
 		public const string BackendMajorVersion = "3";
 		public static readonly Version HwiVersion = new Version("1.0.3");
 		public static readonly Version BitcoinCoreVersion = new Version("0.19.0.1");
+		public static readonly Version LegalDocumentsVersion = new Version(1, 0);
```

Comment:
What about silent updates? Do we want to have that? If yes how to indicate?

Reply:
While I am not sure if it's a good idea to do these at all, if we'd still want to, then it is simple. If we fix a typo or add whitespace, then we don't increase the version. (Note it is not comparing hashes.)